meta-ostro/conf/distro/include/stateless.inc

```diff
@@ -0,0 +1,60 @@
+INHERIT += "stateless"
+
+###########################################################################
+
+# Temporary overrides until Ostro OS is fully stateless.
+
+# This entry here allows everything in /etc because Ostro OS is
+# not actually stateless yet. We merely use the stateless.bbclass
+# to remove files from /etc which get written on the device
+# and thus must be excluded from images and, more importantly,
+# swupd bundles.
+STATELESS_ETC_WHITELIST += "*"
+
+# Empty directories must be kept.
+STATELESS_ETC_DIR_WHITELIST += "*"
+
+###########################################################################
+
+# As an intermediate step towards full stateless Ostro OS, we now
+# treat some files in /etc as conceptually read-only (i.e. neither
```

Comment:
Even when Ostro becomes really stateless, a user might want to add some component which is not.
Having facility in place that can help coping with the statefulness seems like a good idea, compared to requiring the user to fix the component.